lessons/module-3/react-2-the-how.md

```diff
@@ -185,62 +175,55 @@ Let's figure out what should be a component in our app.
 
 Because we want App to hold onto our list of ideas, let's think about how we're going to store that information.
 
-In our first React lesson, we learned about _state_. In order to use component state, we need our component to be a class.
+In our first React lesson, we learned about _state_. In order to use component state, we need to import useState from React.
 
-So: let's import React and the parent Component class, and create our child App component! Think back to your OOP knowhow from Mod 2.
+So: let's import useState from react, and create our child App component!
 
 ```js
-import React, { Component } from 'react';
+import { useState } from 'react';
+function App() {
```

Comment:
I'd love to see a line of whitespace between the import statement and the function declaration.